WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -164,7 +164,7 @@ function() {
 	if (this._objectManager && this._objectManager.reset) {
 		this._objectManager.reset();
 	}
-	this.setScrollWithIframe(this._scrollWithIframe ? DwtControl.CLIP : DwtControl.SCROLL);
```

Comment:
this was added to fix https://github.com/Zimbra/zm-web-client/commit/0caad4f1bc2e29856b0148bd33993bb9ee08ba6b
curious to know how it's working now

Reply:
we added alternative solution in https://github.com/Zimbra/zm-web-client/pull/594 
which is much better that this
1) no additional blank space
2)supports multiple mimes.

Reply:
so decided to remove this line of code. as we don't need this anymore

Reply:
if that was the case, then we should have removed this code in that same PR only, let's not have not needed code in repo to confuse developers

Reply:
Actually i didn’t know about this at that point.

Reply:
This change reintroduced the issue previously fixed by https://github.com/Zimbra/zm-web-client/pull/560, please rollback.